pandas/tests/io/formats/test_printing.py

```diff
@@ -27,7 +26,7 @@ def test_repr_binary_type():
         raw = bytes(letters, encoding=cf.get_option('display.encoding'))
     except TypeError:
         raw = bytes(letters)
-    b = str(compat.bytes_to_str(raw))
+    b = str(raw.decode('utf-8'))
```

Comment:
not a big deal, but is the `str` still necessary?

Reply:
Probably not...could test in a separate commit.